fileErrors.go

```diff
@@ -12,7 +12,13 @@ import (
 )
 
 // ErrFileTooLong is the error given when a file exceeds the maximum possible length
-var ErrFileTooLong = errors.New("file exceeds maximum possible number of lines")
+var (
+	ErrFileTooLong = errors.New("file exceeds maximum possible number of lines")
+	// Similar to FEDACH site
+	ErrToManyRecords = errors.New("there are more than 499 FedACH Participants matching your search criteria. " +
```

Comment:
Should this be Printf formatted with `MaximumRecordsReturned` and other values?

Reply:
Sure

Reply:
I will remove the hard-coding of 499.  Knowing that actual value returned probably not needed.

Reply:
ehh, I'll add it in anyway